// Copyright 2015
// Author: Christopher Van Arsdale

package file.recordio;

message RecordioHeaderProto {
  optional uint64 uncompressed_size = 1;
  optional uint64 compressed_size = 2;

  // TODO: checksum, block encoding, etc.
}



